src/api/app/models/token/release.rb

```diff
@@ -5,27 +5,50 @@ def call(options)
     set_triggered_at
     return unless options[:package]
 
+    # uniq timestring for all targets
+    time_now = Time.now.utc
+
     # FIXME: Take repository and arch into account
     package_to_release = options[:package]
-    manual_release_targets = package_to_release.project.release_targets.where(trigger: 'manual')
-    raise NoReleaseTargetFound, "#{package_to_release.project} has no release targets that are triggered manually" unless manual_release_targets.any?
+    if options[:targetproject].present? && options[:targetrepository].present? && options[:filter_source_repository].present?
```

Comment:
@adrianschroeter I realized that if one of the params is missing we skip this `if` block and execute `release_manually_triggered_targets`. I guess we should return in this case, because a user might just have forgotten to provide one of those params and doesn't want to default to execute the other set manuel release targets

Reply:
I will create a extra commit for this, so you can decide if it makes sense

Reply:
the options also be used to filter pre-defined release definitions from the project... I have not checked, but that might break that (so there may be a usecase to have only some of them defined)

Reply:
@adrianschroeter okay, if thats intended I will remove the commit and adapt the apidocs and specs to reflect that.